pyproject.toml

```diff
@@ -6,11 +6,12 @@ authors = ["SantiagoRugelesSch <ansarusc@gmail.com>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.8"
```

Comment:
This is odd, the statsmodels package says that it should be compatible with 3.8.